BnbnavNetClient/App.axaml

```diff
@@ -48,6 +49,41 @@
     <Style Selector="Button.destructive:pressed">
```

Comment:
IMO, this whole thing should be a part of DayTheme.axaml and NightTheme.axaml

Reply:
This can be resolved later because it doesn't work anyway